### PR TITLE
New L2 ECO and bridge addresses

### PR DESCRIPTION
**Pull request by paged1**
**Description**

Updated ECO bridge addresses for `ethereum`, `optimism`, `goerli`, and `optimism-goerli`.

### CODE REVIEW

**Comment by zainbacchus** [comment]
Thanks we'll get this merged shortly :)

**Comment by paged1** [comment]
Since the schema was updated to support our two L1 bridges (thank you), I've added our `base-goerli` addresses. @roberto-bayardo

**Comment by roberto-bayardo** [comment]
@paged1 could you split out the base-related configs into a separate PR?  We're unfortunately still trying to figure out some of process details around this and I don't want to hold up the optimism updates.

**Comment by paged1** [comment]
@roberto-bayardo I've removed the base goerli addresses and will create a new PR for them

**Comment by roberto-bayardo** [comment]
> @roberto-bayardo I've removed the base goerli addresses and will create a new PR for them

Thanks. @zainbacchus this one is all up to you now.

File: data/ECO/data.json [hunk 1]
```diff
@@ -7,18 +7,32 @@
...
+            "overrides": {
+                "bridge": "0xAa029BbdC947F5205fBa0F3C11b592420B58f824"
+            }
+        },
         "goerli": {
             "address": "0x3E87d4d9E69163E7590f9b39a70853cf25e5ABE3",
             "overrides": {
                 "bridge": {
-                    "optimism-goerli": "0x7a01E277B8fDb8CDB2A2258508514716359f44A0"
+                    "optimism-goerli": "0x9A4464D6bFE006715382D39D183AAf66c952a3e0"
```
[the remark this answers is not in the document]
Comment (paged1) [reply data/ECO/data.json:28]: Thanks, I have verified the implementation at that address.

File: data/ECO/data.json [hunk 2]
```diff
@@ -7,18 +7,32 @@
...
                 }
             }
         },
         "optimism-goerli": {
-            "address": "0x54bBECeA38ff36D32323f8A754683C1F5433A89f",
+            "address": "0xD2f598c826429EEe7c071C02735549aCd88F2c09",
             "overrides": {
-                "bridge": "0x7a01E277B8fDb8CDB2A2258508514716359f44A0"
+                "bridge": "0x6aA809bAeA2e4C057b3994127cB165119c6fc3B2"
```
[the remark this answers is not in the document]
Comment (paged1) [reply data/ECO/data.json:35]: I just verified that address as well.

**Review by tremarkley — APPROVED** [review]

**Pull request merged**